src/password.rs

```diff
@@ -40,6 +41,12 @@ impl Password {
         self
     }
 
+    /// Sets whether empty passwords are allowed (`true``) or not (`false`).
+    pub fn allow_empty(mut self, allow: bool) -> Self {
+        self.allow_empty = allow;
+        self
+    }
+
```

Comment:
I would suggest simplification because an empty password is not allowed by default. Feel free to improve the comment section :)
```suggestion
    /// Allows an empty password (which is not allowed by default).
    pub fn allow_empty(mut self) -> Self {
        self.allow_empty = true;
        self
    }

```

Reply:
@edwardgeorge Other than that, I'm not sure about this feature, what's your use case?

Reply:
Use-case here is when interacting with other systems that allow the user to have empty passwords (the password having already existed in the other system, and not set by using `cliclack`).